### ISSUE TITLE
improve on readme

### ISSUE THREAD

**Issue by HRS8** [HRS8/Portfolio#3]

**Comment by 20481A05K0** [comment]
Hello there can I improve your readme description for your portfolio which has a great impact

**Comment by 20481A05K0** [comment]
Can I work on it

**Comment by HRS8** [comment]
Yes contribution is very much welcomed here,just fork it code the changes and create a pull request

**Comment by 20481A05K0** [comment]
Thank you very much for providing this oppprtunity

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
